pages/agent/v3/signed_pipelines.md

```diff
@@ -0,0 +1,130 @@
+# Signed pipelines
+
+> ⚠️ Warning: Signed Pipelines are experimental, and these docs (and the commands and arguments referenced in them) are subject to change. This is also an opt-in feature - if you're keen to get your hands on it, talk to your sales rep.
```

Comment:
Suggesting someone reach out to a sales rep is new for the docs and makes this sound very enterprise-y. Do smaller self-service customers have a sales rep? Could we handle this through support instead?